llvm/include/llvm/IR/RuntimeLibcalls.h

```diff
@@ -0,0 +1,126 @@
+//===- RuntimeLibcalls.h - Interface for runtime libcalls -------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements a common interface to work with library calls into a
+// runtime that may be emitted by a given backend.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_IR_RUNTIME_LIBCALLS_H
+#define LLVM_IR_RUNTIME_LIBCALLS_H
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/CodeGen/ISDOpcodes.h"
```

Comment:
`llvm/IR` should not depend on `llvm/CodeGen`

Reply:
This should be unused here

Reply:
Oh, I see it's used for the cond code actions. Those should probably be in terms of the IR compare types